Lib/inspect.py

```diff
@@ -3270,6 +3272,9 @@ def _bind(self, args, kwargs, *, partial=False):
                 # before reaching the last parameter before *args.
                 continue
 
+            if param.kind == _POSITIONAL_ONLY:
+                continue
```

Comment:
Tests are passed if remove these lines.

Reply:
Thanks, you're right, once I added a `continue` in a more appropriate place in 69162908a945b998b62a39af9089c6feb8e6b4ff, this became superfluous. Will remove.